packages/manager/apps/pci-kubernetes/src/pages/new/steps/NetworkClusterStep.component.tsx

```diff
@@ -59,7 +85,39 @@ export default function NetworkClusterStep({
         />
       ) : (
         <>
+          {type === DeploymentMode.MULTI_ZONES && (
```

Comment:
We could take benefit from helper :)
```suggestion
          {isMultiDeploymentZones(type) && (
```